src/components/decimal/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from "./decimal.component";
+export type { DecimalProps, CustomEvent } from "./decimal.component";
```

Comment:
Are we sure that the `CustomEvent` type needs to be exported? I'm probably missing something but if not we should only export the minimum that consumers need imo.

Reply:
I think it will be useful for the consumers as they need to type the event in their onChange handlers.

Reply:
Yes that makes sense, thanks. Happy for this to be left in.